examples/jobs/multiple_suites_from_config.py

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env python3
+
+import sys
+
+from avocado.core.job import Job
+from avocado.core.suite import TestSuite
+
+JOB_CONFIG = {'run.test_runner': 'nrunner'}
+
+ORDERLY_CONFIG = {'run.test_runner': 'nrunner',
```

Comment:
Maybe we could omit the runner here.

Reply:
> Maybe we could omit the runner here.

Actually, no, the job level runner selection is not being respected here.  And that's kind of the reason for this example.  I'm opening an issue about a bug here.

Reply:
I did some tests before commenting this and I could not find any "runner' string in test-results log. But maybe we are not logging per test config dict and that is the reason I could not find.